src/drive/ducks/services/components/SuggestionProvider.jsx

```diff
@@ -49,12 +51,12 @@ class SuggestionProvider extends React.Component {
       )
       const files = allDocs.data
 
-      const folders = files.filter(file => file.type === 'directory')
+      const folders = files.filter(file => file.type === TYPE_DIRECTORY)
 
       const normalizedFiles = files
-        .filter(file => file.trashed === false)
+        .filter(file => file.trashed !== true)
```

Comment:
What about `file.trashed && file.trashed === false`? Is not it more expressive?

Reply:
The thing is, if `file.trashed` is `undefined` we want to return `true`, so that the file/folder is not filtered out. `file.trashed && file.trashed === false` would return `false` for a `file.trashed = undefined`.

Reply:
Why do we want to return `true` if `file.trashed === undefined`? That is counterintuitive.
If we extract the filter function into a named function it could be more explicit to understand this counterintuitive behavior:

```js
const notInTrash = file => file.trashed !== true
const normalizedFiles = files.filter(notInTrash)
```

I build a jsbin to illustrate what could be used to return true if value is `undefined`: https://jsbin.com/wurodamupa/edit?js,console

```js
const arr = [ true, false, undefined, null]

console.log(arr.map(val => val))
// [true, false, undefined, null]
console.log(arr.map(val => !val))
// [false, true, true, true]
console.log(arr.map(val => !!val))
// [true, false, false, false]
console.log(arr.map(val => val === false))
// [false, true, false, false]
console.log(arr.map(val => val !== true))
// [false, true, true, true]
```

Reply:
> Why do we want to return true if file.trashed === undefined?

If a file has no `trashed` attribute, it means it's not trashed. AFAIK this is the case for folders at least, who don't have this attribute. Really, the only time a file is trashed is if the `trashed` attribute is `true`.

I'll use a named function as it does make things a bit clearer 👍